.github/workflows/release.yaml

```diff
@@ -24,7 +24,7 @@ jobs:
             run: |
               echo "Pushing Packages"
               cd /work
-              export AUTHOR=$(grep 'service' packages/packages.json | awk -F/ '{print $2}' | head -1)
+              export AUTHOR=$(grep 'service/' packages/packages.json | awk -F/ '{print $2}' | head -1)
```

Comment:
When grepping for "service" the first result is service_registry. I'm adding the slash to enforce that what we are finding is an actual service.